src/transformStateWithClones.js

```diff
@@ -7,7 +7,35 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
+  const x = { ...state };
```

Comment:
[Checklist] [NAMING] - use proper names for object copy

Reply:
fixed